docs/application/dotnet/guides/nui/nui-components/Popup.md

```diff
@@ -5,6 +5,18 @@ A popup can be created using property.
 
 ![PopupProperty](./media/PopupProperty.PNG)
 
+> [!NOTE]
+> Popup is deprecated in API8 and will be removed in API10.

```

Comment:
Global.
How about writing in Tizen Studio version?

Popup is deprecated since Tizen 6.0 and will be removed after two releases.